### PR TITLE
12.0 mig web_dialog_size

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: web_dialog_size/static/src/js/web_dialog_size.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                this.$modal.draggable("destroy");
+            }
+        }
+        return this._super.apply(this, arguments);
+    },
+
+    _extending: function() {
+        var dialog = this.$modal.find('.modal-dialog');
+        dialog.addClass('dialog_full_screen');
+        // dialog.find('.dialog_button_extend').addClass('hidden');
```
Comment (chienandalu) [inline web_dialog_size/static/src/js/web_dialog_size.js]: I'd guess this commented code can be removed :smile:

File: web_dialog_size/README.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+A module that lets the user expand/restore the dialog box size through a button
+in the upper right corner (imitating most windows managers).
+It also adds draggable support to the dialogs.
```
Comment (chienandalu) [inline web_dialog_size/README.rst:30]: It'd be necessary to add a Configure section explaining how to default maximizing in system parameters.

**Review by chienandalu — CHANGES_REQUESTED** [review]
Works perfectly on runbot. Just to small comments

File: web_dialog_size/static/src/js/web_dialog_size.js (hunk 1 above)
Comment (chienandalu) [reply web_dialog_size/static/src/js/web_dialog_size.js]: Maybe you missed this comment

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**